modules/standard/Math.chpl

```diff
@@ -380,6 +381,22 @@ module Math {
       return conj(z);
   }
 
+  /* Returns the complex conjugate of the imaginary argument `z`.
+
+     :rtype: An imaginary number of the same type as `z`.
+  */
+  inline proc conjg(z: imag(?w)): imag(w) {
+    return -z;
+  }
+
+  /* Returns the argument `z`.
+
+     :rtype: A number that is not complex or imaginary of the same type as `z`.
+  */
+  inline proc conjg(z: ?t) where isNumericType(t) {
```

Comment:
proc isNumericType(type t) param return
  isIntegralType(t) || isFloatType(t) || isComplexType(t);


-> this should say where isIntegralType(t) || isFloatType(t)